src/pkg_config_gen.rs

```diff
@@ -51,6 +51,23 @@ impl PkgConfig {
             _ => Vec::new(),
         };
 
+        let libdir = if let Some(subdir) = &capi_config.library.install_subdir {
+            format!("{}{}{}", "${libdir}", MAIN_SEPARATOR, subdir)
```

Comment:
probably we could avoid using `format!` and just append to the path and convert later.

Reply:
Indeed that's easier, done.